k8s/perftest/common/ccpay/ccpay-payment-app.yaml

```diff
@@ -20,10 +20,10 @@ spec:
     java:
       replicas: 2
       useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/payment/api:pr-914-0e18b2e-20210727084929
+      image: hmctspublic.azurecr.io/payment/api:pr-914
```

Comment:
this doesn't look correct?? should be pr-914-3865312-20210818123235 no?

Reply:
It would take latest image after merge